_posts/2022-09-08-storage-write-removal-before-conditional-termination.md

```diff
@@ -87,7 +89,7 @@ To check whether your contract is actually affected, you need to trigger the con
 ## Technical Details
 
 The Yul optimizer step responsible for the bug is the Unused Store Eliminator (abbreviated as `S` in the [optimizer step sequence](https://docs.soliditylang.org/en/latest/yul.html#optimization-step-sequence)). It is meant to remove storage writes that it can determine to be redundant. A storage write is considered redundant, if in all code paths continuing after it either of the following happens:
-- A subsequent write overwrites the value written in the initial write.
+- A subsequent write overwrites the value written in the initial write (without any possibility of the initial write being read back before the overwrite).
```

Comment:
How about this? Too much "write" :P
```suggestion
- The value is overwritten by a subsequent write without ever being read.
```

Also, doesn't the remark about being read back apply to the revert as well?

Reply:
Yeah, it applies to both - I'll try to add something before the items.